ci_build/azure-pipelines/mshost.yaml

```diff
@@ -26,15 +26,9 @@ jobs:
     displayName: Install requirements.txt
 
   - script: |
-      sh ./build.sh
       python setup.py develop
     displayName: Build the library and tests
 
-  - script: |
-      cd out/Linux
-      ./ortcustomops_test
```

Comment:
I understand that onnxruntime can run unit tests for the registered operators. What happens for the other unittests checking internal functions?

Reply:
All unit tests static link to the internal function has been moved into ortcustomops_static_test. So there are two types of test, and the ortcustomops_test tests the whole thing in DLL with ONNXRuntime. So in CI pipeline, the test execution will be  ctest.